network.sh

```diff
@@ -67,11 +68,26 @@ elif [[ -n "${CLUSTER_SUBNET}" ]]; then
 fi
 
 
-if [[ "$IP_STACK" = "v4" ]]
+if [[ "$HOST_IP_STACK" = "v4" ]]
```

Comment:
this works, but maybe could be a good case in which we can use case :)

Reply:
This is consistent with the current IP_STACK code so while arguably a case might be cleaner, such refactoring is probably best handled in a followup?

Reply:
a follow up sounds good